extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -149,7 +150,11 @@ private function renderItem( $type, array $data ) {
 	}
 
 	protected function getThumbnailUrl( $url ) {
-		return VignetteRequest::fromUrl( $url )->scaleToWidth( self::THUMBNAIL_WIDTH )->url();
+		return VignetteRequest::fromUrl( $url )->scaleToWidth(
+			F::app()->checkSkin( 'wikiamobile' ) ?
```

Comment:
I've talked to @hakubo about it. Our PHP doesn't know about mercury and it's gets the same HTML article content as wikiamobile